.evn-sample

```diff
@@ -0,0 +1,5 @@
+DATABASE_URL=postgres://{db_username}:{db_password}@{host}:{port}/{db_name}
```

Comment:
Good work. There's a little typo with your file name. Use sample.env rather than evn.sample

Reply:
Also remove the curly braces, as this can be misleading at times.